contracts/Staking.sol

```diff
@@ -668,13 +668,13 @@ contract OlympusStaking is Ownable {
         @notice trigger rebase if epoch over
```

Comment:
https://github.com/CRE8RDAO/gm-contracts/blob/feat/v1-fantom/contracts/Staking.sol#L620
claim function would be `external`

Reply:
@bejaxer fixed, thanks!